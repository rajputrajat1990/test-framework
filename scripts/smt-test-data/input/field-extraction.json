{"nested": {"user_id": "123", "details": {"name": "john", "email": "john@example.com"}}}
{"nested": {"user_id": "456", "details": {"name": "jane", "email": "jane@example.com"}}}
{"nested": {"user_id": "789", "details": {"name": "bob", "email": "bob@example.com"}}}
