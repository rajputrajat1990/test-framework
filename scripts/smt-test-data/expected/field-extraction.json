{"user_id": "123"}
{"user_id": "456"}
{"user_id": "789"}
